scripts/import_elections.py

```diff
@@ -12,53 +12,55 @@
 # Set up logging
 logging.basicConfig(
     level=logging.INFO,
-    format='%(asctime)s - %(levelname)s - %(message)s',
-    handlers=[
-        logging.FileHandler("election_etl.log"),
-        logging.StreamHandler()
-    ]
+    format="%(asctime)s - %(levelname)s - %(message)s",
+    handlers=[logging.FileHandler("election_etl.log"), logging.StreamHandler()],
 )
 logger = logging.getLogger(__name__)
 
 # Configuration
 CONFIG = {
-    "base_dir": "/home/andrew/projects/cgp_election_data/output/",  # Path to the numbered directories
-    "metadata_path": "/home/andrew/projects/cgp_election_data/output/results-metadata.json",  # Path to metadata JSON file
-    "db_path": "/home/andrew/projects/cgp_election_data/chicago_elections.db",  # Path for the output DuckDB database
+    "base_dir": "output/",  # Path to the numbered directories
```

Comment:
If you follow my revised instructions in the `README` you shouldn't need to specify the paths as long as you `cd` into the folder or open it in VSCode.